lib/parsers/index.ts

```diff
@@ -201,7 +203,7 @@ function processItemGroupForPackageReference(
   includeDev: boolean,
   dependenciesResult,
   propsMap: PropsLookup) {
-  const targetFrameworks: string[] = _.get(packageList, '$.Condition', false) ?
+  const targetFrameworks: string[] = packageList?.$?.Condition ?? false ?
```

Comment:
the density of question marks per line is shocking

Reply:
Yeah, I'm not really a fan. Especially not of the array syntax; `packageList?.$?.[0]?.Condition ?? ...`. But we'll get used to it, I guess.